Client/src/Views/GraphView.tsx

```diff
@@ -8,8 +8,13 @@ import React, { useState } from "react"
 
 import { DatasetsList } from "../Components/Graph/DatasetsList"
 import Graph from '../Components/Graph/Graph'
-import { IDatasetModel } from "../Models/Datasets/IDatasetModel"
+import { ICompleteDatasetEntity } from "../Models/Datasets/ICompleteDatasetEntity"
+import { IGraphDatasetModel } from '../Models/Datasets/IGraphDatasetModel'
 import SearchView from './SearchView'
+import { exampleExportDatasetModel } from '../Models/Datasets/IExportDatasetModel'
+
+//todo this is poorly hardcoded, we need to let user set their own colors, as well as support more than just 4 colors.
```

Comment:
The array itself is just to keep track of the colours according to the array of datasets, once a better mapping is put into place, the set values will be changeable, as well as having more than just 4 datasets per graph, thank you for this observation and comment